docs/HowToGuides/GettingStarted.md

```diff
@@ -272,6 +277,9 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
    - If the build fails:
      See [Troubleshooting build issues](#troubleshooting-build-issues).
 
+   If you would like to build the Swift corelibs, ie libdispatch, Foundation,
+   and XCTest, also on Linux, add the `--xctest` flag to `-build-script`.
+
```

Comment:
Could you instead spell out the full names `swift-corelibs-libdispatch` and so on? This wording is potentially misleading as it makes it sound like one is building Foundation etc. However, what one is actually building is swift-corelibs-foundation etc. This distinction matters because bugs for Foundation (in the Darwin SDKs) should preferably be reported through feedback assistant, whereas bugs for swift-corelibs-foundation should be reported through JIRA.

Reply:
I was unaware of the naming distinction, done.